internal/provider/cluster_resource_test.go

```diff
@@ -468,11 +468,11 @@ func multiRegionServerlessClusterResource(clusterName string) resource.TestStep
 			resource.TestCheckResourceAttr(resourceName, "plan", "SERVERLESS"),
 			resource.TestCheckResourceAttr(resourceName, "state", string(client.CLUSTERSTATETYPE_CREATED)),
 			resource.TestCheckResourceAttr(resourceName, "regions.#", "3"),
-			resource.TestCheckResourceAttr(resourceName, "regions.0.name", "us-west2"),
+			resource.TestCheckResourceAttr(resourceName, "regions.0.name", "europe-west1"),
```

Comment:
For future readers, the regions come back sorted, so we'd have to assert them in a sorted order.